src/main/javascript/src/views/grading/StudentSubmissionGrading.vue

```diff
@@ -21,6 +21,19 @@
         </v-row>
```

Comment:
The total score for the submission should be updateable since that has been removed from the AssignmentScores.vue. See [AssignmentScores.vue updateTotalAlteredGrade](https://github.com/terracotta-education/terracotta/blob/development/src/main/javascript/src/views/grading/AssignmentScores.vue#L148)

Reply:
Added